<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JpkWindow</class>
 <widget class="QDialog" name="JpkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>430</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>681</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QLabel" name="lblFormCode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Kod Formularza</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtFormCode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QRadioButton" name="rbPerson">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Osoba fizyczna</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbNoPerson">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Osoba niefizyczna</string>
     </property>
    </widget>
    <widget class="QFrame" name="framePerson">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>661</width>
       <height>221</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="edtNIP">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtLastName">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtName">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtPhone">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>170</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtEmail">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>90</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblNIP">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>NIP</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblName">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Imie</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblPhone">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>140</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblEmail">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>E-mail</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblLastName">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Nazwisko</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbldateOfBirth">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Data urodzenia</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>110</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Podmiot</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
